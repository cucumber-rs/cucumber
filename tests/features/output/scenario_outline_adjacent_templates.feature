Feature: Outline

  Scenario Outline: foo
    Given foo is <bar1><bar1>
    When foo is <bar2><bar1><bar2>
    Then foo is <bar3><bar2>

    Examples:
      | bar1 | bar2 | bar3 |
      |  0   |  1   |  2   |
