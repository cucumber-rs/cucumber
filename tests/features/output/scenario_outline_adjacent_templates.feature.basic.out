Feature: Outline
  Scenario Outline: foo
   ✔  Given foo is 00
   ✘  When foo is 101
      Step failed:
      Defined: tests/features/output/scenario_outline_adjacent_templates.feature:5:5
      Matched: tests/output.rs:12:1
      Step panicked. Captured output: assertion failed: `(left == right)`
        left: `1`,
       right: `101`
